src/sys/wait.rs

```diff
@@ -25,13 +27,24 @@ libc_bitflags!(
           target_os = "android"))]
```

Comment:
Rather than duplicating this `libc_bitflags`, could you condense these into one? Before each item you can put the proper `#[cfg` so that it still compiles properly. This way we aren't duplicating declarations and comments.